CONTRIBUTING.md

```diff
@@ -0,0 +1,169 @@
+# Contribute to Trinity
+
+This document describes how to contribute to Trinity.
+
+We encourage everyone with knowledge of IOTA technology to contribute.
+
+Thanks! :heart:
+
+<details>
+<summary>Do you have a question :question:</summary>
+<br>
+
+If you have a general or technical question, you can use one of the following resources instead of submitting an issue:
+
+- [**Developer documentation:**](https://docs.iota.org/) For official information about developing with IOTA technology
+- [**Discord:**](https://discord.iota.org/) For real-time chats with the developers and community members
+- [**IOTA cafe:**](https://iota.cafe/) For technical discussions with the Research and Development Department at the IOTA Foundation
+- [**StackExchange:**](https://iota.stackexchange.com/) For technical and troubleshooting questions
+</details>
+
+<br>
+
+<details>
+<summary>Ways to contribute :mag:</summary>
+<br>
+
+To contribute to Trinity on GitHub, you can:
+
+- Report a bug
+- Suggest a new feature
+- Build a new feature
+- Contribute to the documentation
+
+</details>
+
+<br>
+
+<details>
+<summary>Report a bug :bug:</summary>
+<br>
+
+This section guides you through reporting a bug. Following these guidelines helps maintainers and the community understand the bug, reproduce the behavior, and find related bugs.
+
+### Before reporting a bug
+
+Please check the following list:
+
+- **Do not open a GitHub issue for [security vulnerabilities](SECURITY.MD)**, instead, please contact us at [security@iota.org](mailto:security@iota.org).
```

Comment:
```suggestion
- **Do not open a GitHub issue for [security vulnerabilities](SECURITY.MD)**, instead, please submit a report to our [Bugcrowd program](https://bugcrowd.com/iota).
```

Trinity has its own Bugcrowd program, so reports should go there instead of security@iota.org